ReactCommon/react/debug/Android.mk

```diff
@@ -24,8 +24,6 @@ LOCAL_CFLAGS := \
 
 LOCAL_CFLAGS += -fexceptions -frtti -std=c++17 -Wall -llog
 
-LOCAL_LDLIBS := -L$(SYSROOT)/usr/lib -llog
```

Comment:
`-llog` is definitely used by the Android Log functions

Reply:
> `-llog` is definitely used by the Android Log functions

I know, but `LOCAL_LDLIBS` is repeat and `-L$(SYSROOT)/usr/lib` may cause issue [#32595](https://github.com/facebook/react-native/issues/32595#issuecomment-969880095) see the second picture

Reply:
Makes sense thanks for pointing out 👍